packages/beacon-node/test/sim/electra-interop.test.ts

```diff
@@ -224,13 +225,13 @@ describe("executionEngine / ExecutionEngineHttp", function () {
     }
 
     const actualDepositReceipt = payload.depositReceipts[0];
-    if (actualDepositReceipt !== depositReceiptB) {
```

Comment:
These are two objects, to restore previous behavior we need to do a deep equality check, as done previously by `_.isEqual`